test/helpers/kubectl.go

```diff
@@ -35,6 +38,15 @@ const (
 	KubectlCmd    = "kubectl"
 	manifestsPath = "k8sT/manifests/"
 	kubeDNSLabel  = "k8s-app=kube-dns"
+	heapsterLabel = "k8s-app=heapster"
+	ciliumLabel   = "k8s-app=cilium"
+)
+
+var (
+	heapsterYamls = []string{
+		"https://raw.githubusercontent.com/kubernetes/heapster/master/deploy/kube-config/rbac/heapster-rbac.yaml",
```

Comment:
They provide releases, let's use them instead of `master` https://raw.githubusercontent.com/kubernetes/heapster/v1.5.1/deploy/kube-config/rbac/heapster-rbac.yaml